keps/sig-cli/kustomize-subcommand-integration.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Enable kustomize in kubectl
+authors:
+ - "@Liujingfang1"
+owning-sig: sig-cli
+participating-sigs:
+ - sig-cli
+reviewers:
+ - "@liggitt"
+ - "@seans3"
+ - "@soltysh"
+approvers:
+ - "@liggitt"
+ - "@seans3"
+ - "@soltysh"
+editors:
+ - "@pwittrock"
+creation-date: 2018-11-07
+last-updated: 2019-01-17
+status: implementable
+see-also:
+ - "[kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/workflows.md)"
+ - "kustomize-file-processing-integration.md" 
+replaces:
+ - "0008-kustomize.md"
+superseded-by:
+ - n/a
+---
+
+# Enable kustomize in kubectl
+
+## Table of Contents
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Kustomize Introduction](#kustomize-introduction)   
+* [Proposal](#proposal)
+  * [UX](#UX)
+     * [apply](#apply)
+     * [get](#get)
+     * [delete](#delete)
+  * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+  * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Alternatives](#alternatives)
+
+[Tools for generating]: https://github.com/ekalinin/github-markdown-toc
+
+Link to tracking issue: kubernetes/enhancements#633
+
+See [KEP FAQ](kep-faq.md) for "why not as as plugin".
```

Comment:
and `Why should this be part of kubectl?` section too.

Reply:
Done